examples/global-parameters-from-configmap.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: global-simple-config
+  namespace: argo
+  labels:
+    workflows.argoproj.io/configmap-type: parameter
```

Comment:
Currently the value does not matter as long as the key matches so that the configmap informer can take it. Later we can distinguish this from cache configmaps. 

Would you like me to add a single value Enum now or later? I’ll change this to title case.

Reply:
no need to add yet

Reply:
Updated